.github/dependabot.yml

```
@@ -0,0 +1,13 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://docs.github.com/code-security/dependabot/dependabot-version-updates/configuration-options-for-the-dependabot.yml-file
+
+version: 2
+updates:
+  - package-ecosystem: "npm" # See documentation for possible values
+    directories: # Location of package manifests
+      - "/" # package manifest for component itself
+      - "/example" # package manifest for example app
+    schedule:
+      interval: "monthly"
```

Could we add GH Actions updates as well?

```suggestion
      interval: "monthly"
      
  - package-ecosystem: "github-actions"
    directory: "/"
    schedule:
      interval: "monthly"
```

---

That's a great idea! It should be added now. Thanks for the idea.